constants/labels.js

```diff
@@ -0,0 +1,64 @@
+
+const prefixes = {
```

Comment:
I think for consistency good to make all constant identifiers caps (but not sure `PREFIX` is the best naming choice, see below):

```suggestion
const PREFIXES = {
```

Reply:
Done